fe/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -690,7 +692,16 @@ private void handleInsertStmt() throws Exception {
 
             // set to OK, which means the insert load job is successfully submitted.
             // and user can check the job's status by label.
-            context.getState().setOk("{'label':'" + uuid.toString() + "'}");
+            Map<String, String> res = Maps.newHashMap();
```

Comment:
This can make some old client code not compatible. I think if client can get tracking URL from show load, we can not return URL here. 

And this version is deprecated, I think it's better to keep old version's return

Reply:
Change it